### DISCUSSION TITLE
Investigate splitting js into modules for import export

### DISCUSSION THREAD

**Discussion by jmarshvaile** [40dm/gam#15] [General]
received errors when trying to move to a module structure for js

**Comment by jmarshvaile** [comment]
Resolved.

This was a security issue.

When referencing files on local file system, it calls `file///`. The type of request made to call up a module file as done in an import was blocked. In order for the block to not occur, the path needed to be `http`.

Solved by adding Live Server plugin to my visual studio code ide so the file system was attributed to a `http` path instead of a `file///` or `C:` path.

**Comment by jmarshvaile** [comment]
@isaachulvey 

Now that we're using a module structure, you'll need to do your local testing on a live server in order for the file requests to not fail:

https://stackoverflow.com/a/21608670

**Reply by isaachulvey** [reply]
Thanks! I'll check this out
